speechbrain/utils/evaluate_dialogue_state_tracking.py

```diff
@@ -0,0 +1,685 @@
+#!/usr/bin/env python3
+"""
```

Comment:
I feel like the metrics should be extracted into `metric_stats.py` (as mentioned in a later comment) with any dataset-specific quirks as part of the recipes under `meta/`.

If you can think of a clean way to specify all the dataset-specific stuff that isn't centralized that would avoid duplicating the invocation scripts, feel free (in which case it should probably be under `tools` not `utils`).

I couldn't really test this part myself so I refrained from making heavier changes here.